docs/blog/v5-intro.md

```diff
@@ -0,0 +1,167 @@
+
+### No more plugin sidecar
+
+As you might know we have had a python container that have been an [sidecar](https://github.com/grafana-operator/grafana_plugins_init) to the operator that have been in charge of installing the plugins to grafana.
+The sidecar have been badly maintained and to be honest it's just not needed. Instead we use an environment variable to install plugins during startup.
+
+### Helm chart
+
+We have decided to start to manage our own Helm chart. The main reason for this is that we want a simple way of giving you test versions of the operator and many of you use Helm to install the operator.
+
+We still see a big place for the [community Helm chart](https://bitnami.com/stack/grafana-operator/helm) that bitnami is hosting and we hope that they will keep on doing so.
+
+### Namespace mode
+
+Another issues that we have gotten lots of feedback on is [Excessive rbac access](https://github.com/grafana-operator/grafana-operator/issues/604)
+This wasn't something that we could easily fix in v4 but in v5 we have really tried to accommodate this by introducing namespace mode.
+
+In namespace mode you can run the operator to only look for grafana resources in the same namespace that you are in or you can define it to look for a number of specific namespace. By default it will use cluster wide access.
+
+If you install through Helm or Kustomize you will be able to define if you should create cluster roles or only roles.
+Sadly for our Openshift users this won't be possible due to the limitations of OLM, even if you run the operator in namespace mode, which you can. OLM will still use the cluster wide rbac settings.
+
+## v5
+
+So there is still lots to do before version 5 will be released.
+
+There are still a number of features that is missing and we have tried to create issues around this.
+Please have a look in our v5 [milestone](https://github.com/grafana-operator/grafana-operator/milestone/1) to see all open issues.
+
+But we need **your** help, first of all try out version 5.
+
+- Do you like it?
+- Is everything working as you expected?
+- Is there any features missing?
+- Is there an open issue for that feature? Please search for it and if it's not please create an issue.
+
+We probably won't support all the features that we did in v4, due to maintenance burden, but if a missing feature get lots of up votes we might give it a second review.
+
+An extra note about the **do you like it?**, getting feedback is one of the hardest things in open-source.
+Even if "all" you did was to follow the new v5 [installation guide](v5-getting-started.md) we want to know about it, you might feel that you don't have specific insights or thoughts and that is fine.
+But writing one row on slack and saying "hey I installed version v5 and it worked as I expected". Helps us allot since we get to know that people are trying it.
```

Comment:
```suggestion
But writing one row on slack and saying "hey I installed version v5 and it worked as I expected". Helps us a lot since we get to know that people are trying it.
```

Reply:
This have been fixed.